grpc-proto/proto/schema.proto

```diff
@@ -40,6 +38,27 @@ enum ColumnOrderBy {
   DESC = 0x01;
 }
 
+// INDEXES
+
+message CqlIndex {
+  string name = 1;
+  string columnName = 2;
+  bool custom = 3;
+  string customType = 4;
+  map<string, string> options = 5;
+}
+
+// MATERIALIZED VIEWS
+
+message CqlMaterializedView {
+  string name = 1;
```

Comment:
Some other metadata from `system_schema.views` that might be worth exposing here:
- name of the base table
- the custom WHERE clause

I noticed that the persistence API does not expose them so we'd have to add them there too.

Reply:
with the current design, you can only access the MVs from the base table, so you can get the base table name from there.

For custom WHERE clause, this falls into the category of whether we are trying to provide the equivalent of the cqlsh `DESCRIBE` such that the returned schema has the full information required to recreate the schema.

Reply:
> whether we are trying to provide the equivalent of the cqlsh `DESCRIBE` such that the returned schema has the full information required to recreate the schema.

Right, that hasn't been a use case for our persistence API so for, so it's not top priority.